### ISSUE TITLE
Transaction duplication does not apply to today's date

### ISSUE THREAD

**Issue by DevAnyKR** [moneymanagerex/moneymanagerex#6609]
MMEX version:
 - [ ] 1.6.4
 - [ ] 1.7.0
 - [x] 1.7.1 RC1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Date value is not today's date.

Please describe your issue with details.
Add screenshot or other files if needed.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

any transaction view grid.
1. Duplicate Transaction
2. open dialog.
3. Date value is not today's date.

Expected result:
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/e0eef51f-bd8a-4249-bda3-fe938120caf2)
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/898522b8-36b3-4497-b753-be4c84452d68)

Actual result:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/c5f65cdd-9256-4ee8-ba36-c18cdd5aff25)

# Additional information

![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/cfb1ddd8-9fb3-47d9-b07b-ec6a7fd43517)

exist mmexini.db3 in mmex portable. no problem.
use mmexini.db3 in %appdata%. Date value has a problem.

This setting does not seem to be read from the mexini.db3 file. (%appdata%\MoneyManagerEx)
bcz I copied the mexini.db3 file that I used in the existing portable.

**Comment by vomikan** [comment]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/6836805/d13e2cd9-982a-4986-be6a-732a16ec67ba)

**Comment by DevelopmentAnything** [comment]
@vomikan 
I really thought it was only applied when registering for the 'NEW' transaction. I thought it was an exception when copying.
it is recommended that move the copy options to a new option group.

**Comment by n-stein** [comment]
> I really thought it was only applied when registering for the 'NEW' transaction

I agree, we have a specific option for duplicating. The tooltip even says that's what it does:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/6a49f1bf-bcaa-4b87-bba0-e07e18ebafa5)

**Comment by DevelopmentAnything** [comment]
> Are you sure you have removed the mmexini.db3 from the portable folder before running MMEX?

Yes.
I know the cause(vomikan's answer), but it seems that we need to modify the options screen.
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/080fe0c6-3109-4fa3-929f-ea50f24bf312)

I want to apply Last Used for new transactions, and Today Date for transactions that duplicate.
Or reverse.. Today for new transaction, Last Used for duplicate transaction. 😵‍💫🤯😆
